src/patternfly/components/AboutModalBox/about-modal-box.scss

```diff
@@ -2,113 +2,123 @@
 
 .pf-c-about-modal-box {
   // Component variables
-  --pf-c-about-modal-box--sm--grid-template-columns: 5fr 1fr;
-  --pf-c-about-modal-box--lg--grid-template-columns: 70fr 30fr;
-
-  // Modal uses a non-standard background color
-  --pf-c-about-modal-box--BackgroundColor: #{$pf-color-black-1000};
-
-  // This is a special one-off glow for the about modal
-  --pf-c-about-modal-box--BoxShadow: 0 0 100px 0 rgba(255, 255, 255, .05);
+  --pf-c-about-modal-box--BackgroundColor: #{$pf-color-black-1000}; // Modal uses a non-standard background color
+  --pf-c-about-modal-box--BoxShadow: 0 0 100px 0 rgba(255, 255, 255, .05);  // This is a special one-off glow for the about modal
   --pf-c-about-modal-box--ZIndex: var(--pf-global--ZIndex--2xl);
-
-  // Width and height values are determined by design
-  --pf-c-about-modal-box--MaxWidth: #{pf-size-prem(1109px)};
-
-  // Height is optimized for the exact height desired
-  --pf-c-about-modal-box--Height: #{pf-size-prem(686px)};
-
-  // MinHeight is based on all the spacers we know we have, plus some LineHeight to make some space for at least one line of content in each section
-  --pf-c-about-modal-box--MinHeight: calc(var(--pf-c-about-modal-box__brand--PaddingTop) + var(--pf-c-about-modal-box__brand--PaddingBottom) + var(--pf-c-about-modal-box__header--PaddingBottom) + var(--pf-c-about-modal-box__content--MarginTop) + var(--pf-c-about-modal-box__content--MarginBottom) + var(--pf-c-about-modal-box__brand-image--Height) + (1rem * 6 * var(--pf-global--LineHeight--md)));
-
-  // MaxHeight ensures that the modal will not go off the screen and instead the content will scroll
-  --pf-c-about-modal-box--MaxHeight: calc(100vh - var(--pf-global--spacer--xl));
-
-  // Margin left and right make sure there is some space around the modal at the largest breakpoint
-  --pf-c-about-modal-box--MarginLeft: var(--pf-global--spacer--3xl);
-  --pf-c-about-modal-box--MarginRight: var(--pf-global--spacer--3xl);
-
-  // Shared padding values allow us to change common padding to all components once
-  // Some modal components have more specific padding values
+  --pf-c-about-modal-box--Height: 100%;
+  --pf-c-about-modal-box--lg--Height: #{pf-size-prem(762px)}; // Height is optimized for the exact height desired
+  --pf-c-about-modal-box--lg--MaxHeight: calc(100% - var(--pf-global--spacer--xl)); // MaxHeight ensures that the modal will not go off the screen and instead the content will scroll
```

Comment:
Do you need a max-height and max-width if you have set variables from the height and width?

Reply:
I believe so. We want it to be a certain height by default, but not grow beyond `calc(100% - var(--pf-global--spacer--xl));`. And the default width of `calc(100% - (var(--pf-global--spacer--3xl) * 2));` lets it expand on mobile until the max-width of `#{pf-size-prem(1232px)}`. Or maybe I'm misunderstanding your comment :)

Reply:
Ok that makes sense for the width, but maybe for the height it doesn't, since the height is set to an exact value that will not grow.

Reply:
here's a screenshot of what I mean.

What the fixed height does in a tall viewport
<img width="1497" alt="screen shot 2019-01-18 at 1 34 56 pm" src="https://user-images.githubusercontent.com/35148959/51408938-f59da500-1b25-11e9-9a1e-456ed9b9100f.png">

what the max-height does in a short viewport
<img width="1497" alt="screen shot 2019-01-18 at 1 35 08 pm" src="https://user-images.githubusercontent.com/35148959/51408955-03ebc100-1b26-11e9-99df-f739d5540ea4.png">

Reply:
ah ok! I see now